python/feathub/feature_views/tests/test_sliding_feature_view.py

```diff
@@ -47,8 +47,8 @@ def setUp(self):
             path="dummy_source_file",
             data_format="csv",
             schema=Schema(
-                ["id", "fare_amount", "lpep_dropoff_datetime"],
-                [types.Int32, types.Int32, types.Int64],
+                ["id", "id2", "fare_amount", "distance", "lpep_dropoff_datetime"],
```

Comment:
It seems that "id2" is not used in the test cases. Would it be better to remove it?

Reply:
It is used in line 255.